### DISCUSSION TITLE
Delete uploaded audio files - privacy policy

### DISCUSSION THREAD

**Discussion by ces3001** [deepgram/community#659] [General help] [answered]
In your privacy policy, it indicates you keep the uploaded files as long you need to provide the service, which is not very specific. I see no way to delete uploaded audio files, or mark them for removal upon completion of transcription. 

Please provide a setting to:
1) Delete uploaded audio files as soon as possible after transcription (ideally the default)
2) If not, Provide a method for doing so manually.

Thank you for a great service.

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by osamabinsaleem** [comment]
Is it supported?

**Comment by jpvajda** [comment, marked as answer]
@ces3001  @osamabinsaleem  

Thanks for the feedback. We understand why some users of Deepgram might want to delete their audio data after submitting it to us. Currently we don't have a way to do that within the product itself but if you have concerns regarding your data you can contact `security@deepgram.com`  for assistance in this matter.
